v2/client_tweet_recent_search_test.go

```diff
@@ -0,0 +1,46 @@
+package twitter
+
+import (
+	"context"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestClient_TweetRecentSearch(t *testing.T) {
+	type fields struct {
+		Authorizer Authorizer
+		Client     *http.Client
+		Host       string
+	}
+	type args struct {
+		query string
+		opts  TweetRecentSearchOpts
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *TweetRecentSearchResponse
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
Just a reminder that this is here - I'm sure you're working on it or will have another PR.

Reply:
Didn't know github was doing `draft` PRs.  That's cool.

Reply:
Yeah, it is nice to have a built in WIP PR.  I like using them on PRs that have "new" code.  As the reviewers can review the code can use the implementation as it comes.  It helps to not be overwhelmed